src/toolkits/drawing_classifier/dc_data_iterator.cpp

```diff
@@ -25,32 +26,28 @@ namespace {
 
 using neural_net::shared_float_array;
 
-void add_drawing_pixel_data_to_batch(
-    float *next_drawing_pointer, const flex_image &bitmap) {
+void add_drawing_pixel_data_to_batch(float* next_drawing_pointer,
+                                     const flex_image& bitmap) {
   image_util::copy_image_to_memory(
-    /* image input */    bitmap,
-    /* output pointer */ next_drawing_pointer, 
-    /* output strides */ { bitmap.m_width * bitmap.m_channels,
-                           bitmap.m_channels,
-                           1 },
-    /* output shape */   { bitmap.m_height, bitmap.m_width, bitmap.m_channels },
-    /* channel_last */   true);
+      /* image input */ bitmap,
+      /* output pointer */ next_drawing_pointer,
+      /* output strides */
+      {bitmap.m_width * bitmap.m_channels, bitmap.m_channels, 1},
+      /* output shape */ {bitmap.m_height, bitmap.m_width, bitmap.m_channels},
+      /* channel_last */ true);
```

Comment:
I think this change loses some of the readability of having the arguments vertically aligned. Consider adding whitespace inside the comments so that the arguments remain aligned.

Reply:
Done, thanks!